tests/Aspire.Hosting.Azure.Tests/AzureBicepResourceTests.cs

```diff
@@ -1578,7 +1582,6 @@ param principalId string
         Assert.Equal(expectedBicep, storageRolesManifest.BicepText);
```

Comment:
If we're changing this test, then we should change this check to use Verify

Reply:
Sure. Here I only moved the calls that are mutating the service collection while it has already been built.

Reply:
I'd rather update it if we were to change the content of the bicep, or should we also update the whole class if? What was the decision about how we migrate the rest of the tests?

Reply:
> rather update it if we were to change the content of the bicep

When we migrate to Verify that's the only change that should be made in the PR. It would be super hard to diff and track down changes in history if we were to change the behavior and change the verification baseline in the same change.